first commit

yzlucas opened the pull request.

ssylver93 commented on client/wfprev/.vscode/extensions.json:
```diff
@@ -0,0 +1,4 @@
+{
```
.vscode files should be gitignored like in WFNEWS

ssylver93 commented on client/wfprev/src/assets/.gitkeep:
Empty file should be deleted I feel?

ssylver93 commented on client/wfprev/.vscode/launch.json:
```diff
@@ -0,0 +1,20 @@
+{
```
Same for this file

ssylver93 commented on client/wfprev/.vscode/tasks.json:
```diff
@@ -0,0 +1,42 @@
+{
```
Same for this file as well

ssylver93 reviewed (changes requested):
Just a couple of comments on files I think shouldn't be included, looks good otherwise

yzlucas commented:
@ssylver93  thanks, modify the gitingore file to get rid of that folder
@sukhpalp  I have updated to use angular 18 now, the duplicate "db" was an accident, I used it for local test only and I have removed the previous one. The 01_00_00/01 create table thing is for testing the liquibase changes success or not, I think we can modify the changelog.json to not excute it if needed.

yzlucas commented on db/scripts/01_00_00/01/ddl/tables/APP_WFPREV.EXAMPLE_CODE.sql:
Not necessary, I can keep it on my local. I have removed it and update the changelog.json
